### ISSUE TITLE
Text, placeholder text, and images of text meet minimum color contrast standards.

### ISSUE THREAD

**Issue by justpie** [Purdue/purdue-wp-theme#139]
On mobile, the blue (#0088FE) "Back To Top" has a 2.91:1 contrast ratio with the gray (#E9E9E9) background, which falls short of the minimum 4.5:1 required.

**Comment by justpie** [comment]
I am unable to reproduce this. Back to top is black for me?

<img width="678" height="285" alt="Image" src="https://github.com/user-attachments/assets/56a20419-36c2-48da-9ebf-b25acb49881f" />
